### PR TITLE
Fix markdown syntax error in docs [ci-skip]

### PR DESCRIPTION
**Pull request by mehagar**
### Summary

Using backtick instead of single quote was causing a rendering error [here](https://edgeguides.rubyonrails.org/plugins.html#add-an-instance-method).

### CODE REVIEW

File: guides/source/plugins.md [hunk 1]
```diff
@@ -390,7 +390,7 @@ class ActsAsYaffleTest < ActiveSupport::TestCase
 end
 ```
 
-Run the test to make sure the last two tests fail with an error that contains "NoMethodError: undefined method `squawk'",
+Run the test to make sure the last two tests fail with an error that contains "NoMethodError: undefined method 'squawk'",
```
Comment (jonathanhefner) [inline guides/source/plugins.md:393]: Opening backticks can be escaped with `\`: 

```suggestion
Run the test to make sure the last two tests fail with an error that contains "NoMethodError: undefined method \`squawk'",
```

**Review by jonathanhefner — COMMENTED** [review]
Thank you for catching this! :raised_hands: 

I did a quick search (``grep '".+`[A-Za-z0-9_]+'"'" guides/source/*.md``), and didn't find any more occurrences like this. :+1:

File: guides/source/plugins.md (hunk 1 above)
Comment (mehagar) [reply guides/source/plugins.md]: Huh, I didn't realize that the Ruby output was actually a backtick 😯

Comment (jonathanhefner) [reply guides/source/plugins.md]: Indeed, though there is some talk of changing it: https://bugs.ruby-lang.org/issues/16495

**Comment by jonathanhefner** [comment]
@mehagar When you get a chance, would you mind [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?

**Pull request merged**